### ISSUE TITLE
Fails to send SMS with `messagingServiceSid`

### ISSUE THREAD

**Issue by SimplyCorey** [laravel-notification-channels/twilio#62]
A message fails to send when using a `messagingServiceSid` in a notification instead of `from`

This occurs because of this line: https://github.com/laravel-notification-channels/twilio/blob/master/src/TwilioConfig.php#L69

The getFrom errors as the from is not set within the config object.  Instead,  the function call should be

```
    /**
     * Get the default from address.
     *
     * @return string
     */
    public function getFrom()
    {
        if (isset($this->config['from'])) {
            return $this->config['from'];
        }
    }
```

**Comment by SimplyCorey** [comment]
As a workaround for now, we have set from to an empty string to force twilio to use the message service sid.

**Comment by atymic** [comment]
Feel free to PR the fix and i'll check it out.

**Comment by atymic** [comment]
Resolved in #90
